app/page.tsx

```diff
@@ -85,12 +87,12 @@ export default async function Home() {
             backgroundImage: `url(${SeroMap.src})`,
           }}
         >
-          <div className="flex flex-col w-full rounded-md p-4 mb-4 min-h-1/2 h-fit text-background bg-white/90">
-            <div className="w-full h-fit mb-2 lg:mb-0">
+          <div className="grid grid-cols-4 w-full rounded-md p-4 mb-4 min-h-1/2 h-fit text-background bg-white/90">
```

Comment:
This obviously looks bad on smaller screens. I'll fix the responsiveness if we decide this is a direction we're interested in going in.